src/components/flexjar/EmojiRatingFlexjar.tsx

```diff
@@ -68,7 +64,7 @@ export function EmojiRatingFlexjar({ side }: Props) {
     const svar = emojiType && `${emojis[emojiType].score}`;
     if (svar) {
       const body: FlexjarFeedbackDTO = {
-        feedbackId: side,
+        feedbackId: "Arbeidsuførhet",
```

Comment:
Er det uheldig at dette virker som en generisk flexjar-komponent med hardkodet tekst for "arbeidsuførhet"? :thinking:

Reply:
Ja enig i det! Hva med å legge selve filen i arbeidsuforhet mappen? Vet ikke om det er så mye annet i den filen som tilsier at det er en generisk flexjar komponent.

Annen diskusjon da, men denne burde vi kanskje bare slette uansett 🤷‍♂️

Reply:
Flyttet inn flexjar komponentene i mappene til sidene der de brukes 😊

Reply:
Vi har en lapp på å fjerne 8-4 flexjar (https://trello.com/c/pnyvtwoZ/2802-fjerne-flexjar-sp%C3%B8rring-p%C3%A5-8-4) - så kanskje bare gjøre det?

Reply:
Må bare høre med resten av gjengen. Er fort gjort å få fjernet den om vi ønsker det 😁